src/OpenOPC.py

```diff
@@ -260,6 +260,7 @@ def connect(self, opc_server=None, opc_host='localhost'):
       self._group_server_handles = {}
       self._group_handles_tag = {}
       self._group_hooks = {}
+      return connected
```

Comment:
Won't this always return `True`? If it's `False` then an exception is raised on line 244.